### ISSUE TITLE
Any implementation of MixOut Drop Out for Fine Tuning with Regularization

### ISSUE THREAD

**Issue by meethariprasad** [tensorflow/hub#448]
Dear TF Hub Team,

As one of the largest used pretrained language moddel repository, the need for better prebuilt regularizations are more than relevant for TF Hub.

One of the fine recent example is: MIXOUT: EFFECTIVE REGULARIZATION TO FINETUNE
LARGE-SCALE PRETRAINED LANGUAGE MODELS
https://arxiv.org/pdf/1909.11299v1.pdf

Has Mixout been evaluated on fine tuning experiments internally by TF Hub research team?

If so, can we please get the implementation of mixouts?

The reason for better regularization technique is we have seen quite often large scale  Fine Tuning results in to Nan loss, even in the presence of regularizes and dropouts and proper inputs with Sparse Categorical Cross Entropy an I suspect we need better regularizes beyond Dropout and L1-L2 to get stable validation results?

**Comment by vbardiovskyg** [comment]
Hi @meethariprasad, thanks for the suggestion! I think this kind of question would be much better suited for the creators of BERT: https://github.com/google-research/bert.

**Comment by ratthachat** [comment]
This is Pytorch implementation from the author
https://github.com/bloodwass/mixout
